appveyor.yml

```diff
@@ -85,29 +85,15 @@ for:
     - "SET PATH=%PYTHON%;%PYTHON%/Scripts;%PATH%"
     - "python --version"
     - "pip install pdm tox"
-    - "pdm install --no-self"
+    - "pdm install -v --dev --no-self"
```

Comment:
add the `--dev` flag to install the dev dependencies that are needed to run the scripts (`requests`)